ivy.xml

```diff
@@ -15,7 +15,8 @@
  * limitations under the License.
 -->
 <!DOCTYPE ivy-module [
-  <!ENTITY maven.version "2.0">
+  <!ENTITY maven.version "3.5.0">
```

Comment:
@slachiewicz Any particular reason? Why is 3.5.0 with Resolver 1.0.3 not good enough for this plugin?

Reply:
Will be good. My proposal is just latest available version from this line. We voted some time ago to say that is minimal "supported" for plugins for now.

Reply:
Sure for Maven owned plugins this is fine, but this is a 3rd party one, which just requires 3.5.0 because the API changed fundamentally. It is not the obligation of 3rd party plugins to encourage Maven upgrades to latest from line.

Reply:
Yes, true. No obligation.
But 3.5.0 is not supported by us (Maven Core Developers) - nothing more.

Reply:
This is just the version we compile against, we never run any code from it. As bugfix releases may never change method signatures, this is safe.